src/lib.rs

```diff
@@ -27,4 +27,7 @@ pub mod tasks;
 /// Utilities.
 pub mod utils;
 
+/// Anonymous crate dependency imports.
+use metrics as _;
+use metrics_exporter_prometheus as _;
```

Comment:
you can remove these as deps, as `metrics` is re-exported from init4-bin-base, and the exporter is set up for you in bin-base